triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceMain.scala

```diff
@@ -168,18 +168,25 @@ object Server {
           }
         )
       },
-      // List triggers currently running for the given party
       get {
-        path("list") {
-          entity(as[ListParams]) { params =>
-            {
-              val triggerList =
-                triggersByParty.getOrElse(params.party, Set()).map(_.toString).toList
-              val result = JsObject(("triggerIds", triggerList.toJson))
-              complete(successResponse(result))
+        // Convenience endpoint for tests (roughly follow
+        // https://tools.ietf.org/id/draft-inadarei-api-health-check-01.html).
+        concat(
+          path("health") {
+            complete((StatusCodes.OK, JsObject(("status", "pass".toJson))))
```

Comment:
Sorry I just saw this, but this response format is not quite right. The `"status"` field is supposed to mirror the status code, and there should be a `"result"` field with whatever successful response body you want, i.e. `{"status": 200, "result": value}`. I suggest using the `successResponse` helper function as in the `list` command, where you pass in the JSON `value` you want (could be the string "pass" in this case).

Reply:
> Sorry I just saw this, but this response format is not quite right. The `"status"` field is supposed to mirror the status code, and there should be a `"result"` field with whatever successful response body you want, i.e. `{"status": 200, "result": value}`. I suggest using the `successResponse` helper function as in the `list` command, where you pass in the JSON `value` you want (could be the string "pass" in this case).

This case intentionally breaks with our internal conventions in order to conform to the referenced RFC (internet standard proposal).